templates/routing-typescript/src/pages/pageOne.tsx

```diff
@@ -37,6 +37,9 @@ export const PageOne = (): JSX.Element => {
                     icon={<Event fontSize={'inherit'} />}
                     title={'Coming Soon'}
                     description={'Replace this page with your own content'}
+                    placeholder={undefined}
```

Comment:
Don't add these useless props...use the workaround for the issue in @types/react used here: https://github.com/etn-ccis/blui-icons/blob/cb6b0a2fdaec624e4f6bf14c71dc2340239442fa/demos/react-demo/src/index.tsx#L21